### PR TITLE
#cm_17

### PR DESCRIPTION
**Pull request by yaroslava-khlystun**

### CODE REVIEW

**Comment by yaroslava-khlystun** [comment]
#cm_17

File: app/scripts/modules/Registration.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var React = require('react');
+
+var Registration = React.createClass({
+  render: function() {
+    return (
+      <form className="registration" role="form" onChange={this.handleChange}>
+        <div>
+          <div>
+            <input className="registration__input" type="text" name="fName" ref="firstName" placeholder="First Name"/>
```
Comment (tkharuk) [inline app/scripts/modules/Registration.js:9]: why double nested empty 'div'? O.o

File: app/scripts/modules/Registration.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          </div>
+        </div>
+        <div>
+          <div>
+            <input className="registration__input" type="password" name="password" ref="password" value="" placeholder="Password"/>
+          </div>
+        </div>
+        <div>
+          <div className="item-cont">
+            <input className="registration__input" type="password" name="confirmPassword" ref="confPassword" value="" placeholder="Confirm Password"/>
```
Comment (tkharuk) [inline app/scripts/modules/Registration.js:29]: no need for confirmation

File: app/scripts/modules/Registration.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+      <form className="registration" role="form" onChange={this.handleChange}>
+        <div>
+          <div>
+            <input className="registration__input" type="text" name="fName" ref="firstName" placeholder="First Name"/>
+          </div>
+        </div>
+        <div>
+          <div>
+            <input className="registration__input" type="text" name="lName" ref="lastName" placeholder="Last Name"/>
```
Comment (tkharuk) [inline app/scripts/modules/Registration.js:14]: no need for Last name. use 1 field to have both first and last name e.g 'John Doe'

File: app/styles/modules/_registration.scss [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        display: inline-block;
+        padding: 15px 20px;
+        margin-bottom: 1px;
+        color: #fff;
+        text-align: center;
+        vertical-align: middle;
+        cursor: pointer;
+        opacity: 1;
+        border: none;
+        background-color: #C89F1C;
```
Comment (tkharuk) [inline app/styles/modules/_registration.scss:34]: use variables!

File: app/styles/modules/_registration.scss [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            box-shadow: inset 0 -0.15em 0 rgba(0, 0, 0, 0.2);
+            background-color: #283645;
+        }
+    }
+
+    input[type=text], input[type=password], input[type=email] {
+        border-style: solid;
+        border-width: 2px;
+        border-color: #bdc3c7;
+        -webkit-transition: border-color 0.5s ease-in-out !important;
```
Comment (tkharuk) [inline app/styles/modules/_registration.scss:52]: never use important!

File: app/styles/modules/_registration.scss [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        transition: border-color 0.5s ease-in-out !important;
+        border-radius: 3px;
+    
+        padding: 10px 10px 9px 2.3em;
```
Comment (tkharuk) [inline app/styles/modules/_registration.scss:56]: mixed px and em

File: app/styles/modules/_registration.scss [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    
+        outline: none;
+        -o-box-sizing: border-box;
```
Comment (tkharuk) [inline app/styles/modules/_registration.scss:59]: no need in browser prefixes - http://caniuse.com/#feat=css3-boxsizing

File: app/styles/modules/_registration.scss [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+@import '../mixins';
+
+.registration {
+    margin: 150px auto;
+    width: 80%;
+
+    &>div {
+        margin: 25px;
+        position: relative;
+        xmargin-left: 25px;
```
Comment (tkharuk) [inline app/styles/modules/_registration.scss:11]: no such property 'xmargin'

File: app/styles/modules/_registration.scss [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    .registration__input {
+        width: 60%;
+        margin: auto;
+        font-size: 1em;
+        line-height: 1.231;
+    }
+
+    input[type=submit] {
```
Comment (tkharuk) [inline app/styles/modules/_registration.scss:22]: attribute value should be wrapped in quotes - https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

**Comment by tkharuk** [comment]
also, set your git preferences so we can see who have commited.
currently, it looks like this:

Registration form. w/o validation&firebase
unknown authored 2 hours ago

**Pull request closed without merging**